experimental/segmenter/src/lstm_bies.rs

```diff
@@ -157,11 +157,12 @@ impl<'l> Lstm<'l> {
                 .collect::<Vec<usize>>()
                 .windows(2)
                 .map(|chunk| {
-                    self.return_id(
-                        input
-                            .get(*chunk.get(0).unwrap_or(&0)..*chunk.get(1).unwrap_or(&input.len()))
-                            .unwrap_or(input),
-                    )
+                    let range = if let [first, second, ..] = chunk {
+                        *first..*second
+                    } else {
+                        unreachable!()
```

Comment:
~Issue: I think this is panicky where it wasn't panicky before.~

Clarified: it is unreachable because of the `.windows(2)`